sporkfish/searcher/searcher_factory.py

```diff
@@ -13,7 +13,7 @@ class SearcherFactory:
     """
 
     @staticmethod
-    def _build_moveorder(order_type: MoveOrderMode) -> MoveOrder:
+    def _build_moveorder(searcher_cfg: SearcherConfig) -> MoveOrder:
```

Comment:
What is the reason for this change? e.g. is it more user friendly, or is it some subtle performance thing?

[edit] ah ok, I see you need extra config info as part of this change...sorry was blind